src/components/IndividualArticle.jsx

```diff
@@ -0,0 +1,34 @@
+import { useParams } from "react-router-dom";
+import { fetchIndividualArticle } from "../utils/apicalls";
+import { useEffect, useState } from "react";
+
+
+export const IndividualArticle = () => {
+const {article_id} = useParams()
+const [article, setArticle] = useState([])
+const [loading, setLoading] = useState(false)
+const number = Number(article_id)
+useEffect(() => {
+
+    setLoading(true)
+
+    fetchIndividualArticle(number).then((response) => {
+        setLoading(false)
+        setArticle(response.data.article)
+})
+}, [article_id])
+
+if(loading){return <p>Loading...</p>}
+
+const date = new Date(article.created_at).toDateString()
+
+return (<div className='individual-article'>
```

Comment:
is `<div>` the most suitable semantic html element you could have used here? 
What about `<section>` or `<article>`.